services/supabase/migrations/20260827000002_create_story_images_bucket.sql

```
@@ -0,0 +1,29 @@
+  TO authenticated
+  WITH CHECK (bucket_id = 'project_story_images');
+
+-- Allow the public to read story images (needed for embedding in project pages).
+CREATE POLICY "Story images are publicly readable"
+  ON storage.objects
+  FOR SELECT
+  TO public
+  USING (bucket_id = 'project_story_images');
+
+-- Allow authenticated users to delete their own story images.
+CREATE POLICY "Authenticated users can delete story images"
```

SQLFluff (4.3.0)

[error] Do not use special characters in identifiers.

(RF05)